integrations/Appcues/browser.js

```diff
@@ -0,0 +1,77 @@
+import logger from "../../utils/logUtil";
+import ScriptLoader from "../ScriptLoader";
+
+class Appcues {
+  constructor(config) {
+    this.accountId = config.accountId;
+    this.apiKey = config.apiKey;
+    this.name = "APPCUES";
+    //this.sendToAllDestinations = config.sendToAll;
+  }
+
+  init() {
+    logger.debug("===in init Appcues===");
+    ScriptLoader(
+      "appcues-id",
+      `https://fast.appcues.com/${this.accountId}.js`
+    );
+  }
+
+  isLoaded() {
+    logger.debug("in appcues isLoaded");
+    return !!window.Appcues;
+  }
+
+  isReady() {
+    logger.debug("in appcues isReady");
+    // if (this.sendToAllDestinations) {
+    //   window.Appcues.on("all", function(eventName, event) {
+    //     window.rudderanalytics.track(eventName, event, {
+    //       integrations: {
+    //         All: true,
+    //         APPCUES: false
+    //       }
+    //     });
+    //   });
+    // }
+    return !!window.Appcues;
+  }
+
+  identify(rudderElement) {
+    const {
+      traits
+    } = rudderElement.message.context;
+    const {
+      userId
+    } = rudderElement.message;
+    if (userId) {
+      window.Appcues.identify(userId, traits);
+    }
+  }
+
+  track(rudderElement) {
+    const eventName = rudderElement.message.event;
+    let {
+      properties
+    } = rudderElement.message;
+    if (eventName) {
+      window.Appcues.track(eventName, properties);
```

Comment:
what happens if properties --> null?

Reply:
It would work fine even if the properties are null